plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -301,7 +329,10 @@ read_write::push_block_results read_write::push_block(const read_write::push_blo
 }
 
 read_write::push_transaction_results read_write::push_transaction(const read_write::push_transaction_params& params) {
-   auto ptrx = db.push_transaction(params);
+   if (skip_flags & chain_controller::skip_transaction_signatures) {
```

Comment:
rather than duplicating the bitwise logic, this should call chain_plugin::skip_transaction_signatures.